README.md

```diff
@@ -1,12 +1,49 @@
 # HealthCards
 
-Creating verifiable credentials with FHIR using [SMART Health Cards](https://smarthealth.cards/)
+![Build](https://github.com/dvci/health_cards/actions/workflows/ruby.yml/badge.svg)
+![Maintainability](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/maintainability)
+![Test Coverage](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/test_coverage)
 
-## Developing
+This repository includes a Ruby gem for [SMART Health Cards](https://smarthealth.cards/) **and** a reference implementation for the [Vaccine Credential Initiative Implementation Guide](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/). Go to the [Health Card Gem](#health-card-gem) to read about the Ruby library or go to the [Reference Implementation](#reference-implementation) section to try a ready-to-use rails application. 
 
-Use `bin/setup` to try things out on the command line.
+## Reference Implementation
 
-## Installation
+The reference implementation here is a Ruby on Rails application that showcases both the Issuer side for creating SMART Health Cards and the Verifier side for confirming someone's vaccination status or laboratory test result. There are three endpoints for the representation and communication of a SMART Health Card. One is a QR Code that may be digital (in PDF format) or printed for physical handout. On the verifier side the QR code may be scanned and verified by a webcam. The second endpoint is a `*.smart-health-card` file that may be digitally transferred and used by other SMART-compatable applications. The third endpoint is a `$health-card-issue` FHIR operation that other FHIR v4.0.1 compatable applications can use as a microservice API call.
+
+### Dependencies
+ - Ruby 2.7 (prior versions may work but are not tested for)
+ - [Bundler](https://bundler.io)
+ - [Yarn](https://yarnpkg.com)
+ - Rails 6.1
+
+### Quick Start
+
+Clone and cd into this repository:
+
+`git clone https://github.com/dvci/health_cards.git && cd health_cards`
+
```

Comment:
Since this is meant to be instructional let's break out the `git clone` and `cd` onto separate lines to make it clear what's going on. i.e.:

```shell
git clone https://github.com/dvci/health_cards.git
cd health_cards
```

Reply:
fixed